build/devops_gated.yml

```diff
@@ -0,0 +1,312 @@
+name: $(BuildID)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
```

Comment:
@dcristoloveanu How do we setup the gated builds for this repo?

Reply:
@mattdurak @parth21999 I can create a build in our folder in msazure

Reply:
@mattdurak @parth21999 Done, setup.